docs/REST API/rest-api.md

```diff
@@ -9238,6 +9238,7 @@ Returns information about the specified software. By default, `versions` are sor
     "id": 12,
     "name": "Falcon.app",
     "bundle_identifier": "crowdstrike.falcon.Agent",
+    "available_fleet_maintained_app_id": null,
```

Comment:
```suggestion
    "available_apps": {
        "fleet_maintained_app": {
            "id": 4
        },
        "app_store_app": {
            "app_store_id": "8675309"
        }
    }
```

Tweaking this for forward-compat with #27592 (minimal extra effort to include this in the API work for this feature, allowing 27592 to be FE-only). In this particular case we'd have neither an App Store app nor an FMA so this part of the payload would be:

```json
    "available_apps": {
        "fleet_maintained_app": null,
        "app_store_app": null
    }
```

/cc @noahtalerman

Reply:
Yeah, this'll be 4.68.

We wound up not getting to this in today's design review, but given that the above API design makes sense I'll take as an action item getting a new PR for the above up and pointed at 4.68 by design review tmw.